src/tests/nativeaot/nativeaot.csproj

```diff
@@ -1,8 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <ItemGroup>
-      <MergedWrapperProjectReference Include="*/**/*.??proj" Exclude="*/**/Library.csproj" />
-    </ItemGroup>
-  
-    <Import Project="$(TestSourceDir)MergedTestRunner.targets" />
-  </Project>
+  <PropertyGroup>
+    <CLRTestExitCode Condition="'$(CLRTestExitCode)' == ''">100</CLRTestExitCode>
```

Comment:
This should already be set by `Common\CLRTest.Execute.targets`, why does it need to be duplicated?

Reply:
removed.